openbb_terminal/economy/investingcom_model.py

```diff
@@ -33,6 +33,229 @@
 ]
 IMPORTANCES = ["high", "medium", "low", "all"]
 
+# Commented countries either have no data or are not correctly formatted in investpy itself
+MATRIX_COUNTRIES = {
+    "G7": [
+        "United states",
+        "Canada",
+        "Japan",
+        "Germany",
+        "France",
+        "Italy",
+        "United Kingdom",
+    ],
+    "PIIGS": [
+        "Portugal",
+        "Italy",
+        "Ireland",
+        "Greece",
+        "Spain",
+    ],
+    "EZ": [
+        "Austria",
+        "Belgium",
+        "Cyprus",
+        # "Estonia",
+        "Finland",
+        "France",
+        "Germany",
+        "Greece",
+        "Ireland",
+        "Italy",
+        # "Latvia",
+        # "Lithuania",
+        # "Luxembourg",
```

Comment:
Is there a reason certain ones are commented out instead of just not being included?

Reply:
Yes, these countries belong to the group and Investing.com have some of them available in the website. 

There seems to be an issue with investpy library when getting the data. So I just commented them for now to avoid warning and not forget to include them when it is fixed.

Reply:
Do u think we should keep them all so it works automatically later? Downside is the warning..

Reply:
>  @jmaslek do u think we should keep them all so it works automatically later? Downside is the warning..

Reply:
I will leave that up to you.  Can always leave it as a TODO: to check when they are available.